packages/website/src/Store.ts

```diff
@@ -26,7 +26,7 @@ const loadFromSessionStorage = () => {
 };
 
 const persistedState =
-    process.env.NODE_ENV === 'development' && (window as any).loaded ? loadFromSessionStorage() : undefined;
+    process.env.NODE_ENV === 'development' && (globalThis as any)?.loaded ? loadFromSessionStorage() : undefined;
```

Comment:
window is not defined on the first load because Nextjs is pre-rendering in a non-browser environment (node). However `globalThis` which later on points to `window` is defined both in node and in browser, so it fixes the issue.

Reply:
Without this fix:
![image](https://user-images.githubusercontent.com/35579930/167897472-d7890c37-0aec-4646-8d92-8351f3272d04.png)